crates/aptos-dkg/src/pvss/das/weighted_protocol.rs

```diff
@@ -538,3 +538,125 @@ impl Transcript {
         }
     }
 }
+
```

Comment:
Why are you doing it here only for `das` instead of in `tests/pvss.rs`?

Reply:
oops, fixed